test/integration/eth-simulator.ts

```diff
@@ -81,26 +80,67 @@ test('ETH: send allowed eth63', async t => {
       t.end()
     }
   }
-  util.twoPeerMsgExchange(t, capabilities, opts)
+  util.twoPeerMsgExchange(t, opts, cap)
+}
+
+test('ETH: should use latest protocol version on default', async t => {
+  sendWithProtocolVersion(t, 64)
 })
 
-test('ETH: send allowed eth62', async t => {
-  const cap = [devp2p.ETH.eth62]
+test('ETH: should work with allowed eth64', async t => {
+  sendWithProtocolVersion(t, 64)
```

Comment:
This test is exactly the same as the one above?

Reply:
Yes, the test cases are semantically different and I wanted to have it in a way that someone updating to a new protocol version (ETH/65) is triggered to on that occasion update the first test case.